### DISCUSSION TITLE
Can I make the preview window bigger, so that I can take a screenshot with clear text?

### DISCUSSION THREAD

**Discussion by utkarshxs** [markmap/markmap#216] [General]
![image](https://github.com/markmap/markmap/assets/78845394/480df63e-0df1-4907-a744-d553dd4ffa26)
I want to make a flowchart and paste it in my notes in Evernote. Since evernote doesn't offer Markdown/markmap embeds, I am using screenshots.

**Comment by gera2ld** [comment]
Currently the only way is to save your Markdown in a gist and open the gist through "Open URL".

Or you can export it as an HTML and open it in your browser.
